<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drone Flight Recorder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Serial device</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="comboBoxSerialSelect"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonSerialRefresh">
         <property name="text">
          <string>Re&amp;fresh</string>
         </property>
         <property name="shortcut">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonSerialConnect">
         <property name="text">
          <string>&amp;Connect</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Data interaction</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonRecord">
         <property name="text">
          <string>&amp;Record</string>
         </property>
         <property name="shortcut">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonPlay">
         <property name="text">
          <string>&amp;Play</string>
         </property>
         <property name="shortcut">
          <string>P</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_data"/>
    <addaction name="action_Save_data"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAuthor"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="actionClear_play_data"/>
    <addaction name="actionClear_record_data"/>
    <addaction name="actionSet_play_interval"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Data"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_data">
   <property name="text">
    <string>&amp;Open data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_data">
   <property name="text">
    <string>&amp;Save data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClear_play_data">
   <property name="text">
    <string>Clear &amp;play data</string>
   </property>
   <property name="shortcut">
    <string>Shift+P</string>
   </property>
  </action>
  <action name="actionClear_record_data">
   <property name="text">
    <string>Clear &amp;record data</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionSet_play_interval">
   <property name="text">
    <string>Set play &amp;interval</string>
   </property>
   <property name="shortcut">
    <string>Shift+I</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
